frontend/include/chpl/uast/BuilderResult.h

```diff
@@ -34,6 +34,24 @@
 
 #include "llvm/ADT/DenseMap.h"
 
+#define CHPL_ID_LOC_MAP_INNER(ast__, location__) \
+  id##ast__##To##location__##Location_
```

Comment:
Why do you need the 'inner' macro here?

Reply:
Otherwise the concatenated macro tokens won't expand.